core/handler/device/store.go

```diff
@@ -39,18 +41,22 @@ func NewRedisDeviceStore(client *redis.Client, prefix string) *RedisDeviceStore
 	for v, f := range migrate.DeviceMigrations(prefix) {
 		store.AddMigration(v, f)
 	}
+	s, _ := client.Get(prefix + ":" + redisDevicePrefix + ":whitelist").Result()
+	l := strings.Split(s, ":")
```

Comment:
No idea on how to handle an error there... apart for fmt.Printf(err)

The whitelist is stored as a simple string. Each key is separated by ":" so simply split it into a slice will do. Using set is overkill for this.